rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/RMRest.java

```diff
@@ -611,6 +605,26 @@ public String getNodeMBeanHistory(@HeaderParam("sessionid") String sessionId,
         return rmProxy.getNodeMBeanHistory(nodeJmxUrl, objectName, attrs, range);
     }
 
+    @Override
+    @GET
+    @GZIP
+    @Produces("application/json")
+    @Path("nodes/mbean/history")
+    public String getNodesMBeanHistory(@HeaderParam("sessionid") String sessionId,
```

Comment:
The typing is not aligned with the declaration in RMRestInterface interface. Please confirm it's OK.

Reply:
You are right. It has to be Object as in the declaration in RMRestInterface interface.